src/content/docs/integrations/amazon-integrations/aws-integrations-list/aws-vpc-flow-logs-monitoring-integration.mdx

```diff
@@ -16,6 +16,10 @@ redirects:
   The New Relic VPC Flow Logs integration can only process logs in AWS's default format. For more information on VPC Flow Logs formatting, see [Amazon's VPC Flow Logs documentation](https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs.html#flow-log-records).
 </Callout>
 
+<Callout variant="important">
```

Comment:
I'd move this information to the requirements section, so users know about this before actually connecting their account. So I'd add the following paragraph after the one we already have:
`For you to see the VPC logs, your AWS account must be integrated with the Metric streams mode. If your account is integrated using the API polling mode, you cannot receive to the VPC logs.`